spec/unit/matrixrtc/MatrixRTCSession.spec.ts

```diff
@@ -869,14 +874,24 @@ describe("MatrixRTCSession", () => {
                         .mockReturnValue(makeMockRoomState([membershipTemplate], mockRoom.roomId));
                     sess.onRTCSessionMemberUpdate();
 
-                    jest.advanceTimersByTime(10000);
+                    jest.advanceTimersByTime(3000);
```

Comment:
Should this also be `KEY_DELAY` or it something else that's coincidentally also 3s?